sdk/core/System.ClientModel/src/Message/PipelineResponseHeaders.cs

```diff
@@ -0,0 +1,189 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ClientModel.Internal;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Runtime.CompilerServices;
+
+namespace System.ClientModel.Primitives;
+
+internal class PipelineResponseHeaders : PipelineMessageHeaders
```

Comment:
this class is specific to HttpClient stack. I think it would be good if the name reflected it.

Reply:
It's internal, so I'm fine either way!  Will fix.